apis/submariner/v1alpha1/submariner_types.go

```diff
@@ -124,8 +124,54 @@ type SubmarinerList struct {
 	Items           []Submariner `json:"items"`
 }
 
+// BrokerSpec defines the desired state of Broker
+// +k8s:openapi-gen=true
+type BrokerSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	Components                  []string `json:"components,omitempty"`
+	DefaultCustomDomains        []string `json:"defaultCustomDomains,omitempty"`
+	GlobalnetCidrRange          string   `json:"globalnetCidrRange,omitempty"`
```

Comment:
We've been trending from `Cidr` -> `CIDR` elsewhere... 

```suggestion
	GlobalnetCIDRRange          string   `json:"globalnetCIDRRange,omitempty"`
```

Reply:
Ah yes, fixed.